compiler/src/dotty/tools/dotc/transform/GenericSignatures.scala

```diff
@@ -5,16 +5,19 @@ package transform
 import core.Annotations._
 import core.Contexts._
 import core.Phases._
+import core.Decorators.*
 import core.Definitions
 import core.Flags._
 import core.Names.Name
 import core.Symbols._
 import core.TypeApplications.{EtaExpansion, TypeParamInfo}
-import core.TypeErasure.{erasedGlb, erasure, isGenericArrayElement}
+import core.TypeErasure.{erasedGlb, erasure, fullErasure, isGenericArrayElement}
 import core.Types._
 import core.classfile.ClassfileConstants
 import SymUtils._
 import TypeUtils._
+import config.Printers.transforms
+import reporting.trace
```

Comment:
Imports w.r.t. debugging can be dropped

Reply:
I'd love to never have to import them, so this is 1 more file that won't have to happen...